cmd/util/ledger/util/util.go

```diff
@@ -24,25 +23,37 @@ func newRegisterID(owner []byte, key []byte) flow.RegisterID {
 
 type AccountsAtreeLedger struct {
 	Accounts environment.Accounts
+	temp     map[string][]byte
```

Comment:
I added because a test was failing. Looking into it again, it is only `TestFixSlabsWithBrokenReferences` that fails, and it only performs reads for temporaries, no writes.

In c24a98ee0a677b70bef8a7506108e9fdbd541c85 I removed the support for writing temporaries, but needed to keep support for reading them without producing an error.

I'm not 100% if this is the right place though, or that logic should be moved up/down the stack.